src/schemathesis/runner/__init__.py

```diff
@@ -66,6 +66,8 @@ def execute_from_schema(
         yield events.Initialized(results=results, schema=schema, checks=checks, hypothesis_settings=settings)
 
         for endpoint, test in schema.get_all_tests(single_test, settings):
+            if not endpoint.is_valid:
```

Comment:
In this case, we should not run the test later on - we can put this condition to `try/except`:

```
try:
    if endpoint.is_valid:
        test(session, base_url, checks, result)
        status = Status.success
    else:
        status = Status.invalid
except AssertionErrpr:
    ...
```